src/services/item/plugins/publication/validation/repositories/ItemValidationGroup.ts

```diff
@@ -22,42 +24,33 @@ export const ItemValidationGroupRepository = AppDataSource.getRepository(
     }
 
     return ivg;
-  },
+  }
 
   /**
    * Get last item validation group for given item id
    * @param {string} itemId id of the item being checked
    */
-  async getLastForItem(itemId: string): Promise<ItemValidationGroup> {
+  async getLastForItem(itemId: string): Promise<ItemValidationGroup | null> {
     // return this.createQueryBuilder('iVG')
```

Comment:
Can we check that `itemId` is not `undefined`? Otherwise typeorm could return everything... (I know the type is string but well...)